contributing.md

```diff
@@ -0,0 +1,165 @@
+# 🤝***First time contributing? We will help you out.*** 👍
+Refer to the following articles on the basics of Git and Github and can also contact the Project Mentors, in case you are stuck:
+
+
+- [Getting started with Git and GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github)
+- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
+- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+
+***If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).***
+
+## 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+
+- Take a look at the Existing [Issues] (https://github.com/anitab-org/documentation/issues) or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a Branch for any Issue that you are working upon.
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add Screenshots to help us know what this Script is all about.
+
+
+## ⭐HOW TO MAKE A PULL REQUEST:
+**1.** Fork [this](https://github.com/anitab-org/documentation) repository.
+Click on the <a href="https://github.com/anitab-org/documentation"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+
+**2.** Clone the forked repository.
+
+```bash
+git clone https://github.com/<your-github-username>/documentation
+```
+
+**3.** Navigate to the project directory.
+
+```bash
+cd documentation
+```
+**5.** Create a new branch:
+```bash
+git checkout -b YourBranchName
+```
+
+**6.** Make changes in source code.
+**7.** Stage your changes and commit
+
+```bash
+git add .
+```
+```bash
+git commit -m "<your_commit_message>"
+```
```

Comment:
This should be written this way

**6.** Make changes in source code.

```bash
git add .
```

**7.** Stage your changes and commit

```bash
git commit -m "<your_commit_message>"
```